internal/dump/ingestpipelines.go

```diff
@@ -6,122 +6,17 @@ package dump
 
 import (
 	"context"
-	"encoding/json"
-	"fmt"
-	"io"
-	"net/http"
-	"slices"
 
 	"github.com/elastic/elastic-package/internal/elasticsearch"
+	"github.com/elastic/elastic-package/internal/elasticsearch/ingest"
 )
 
-// IngestPipeline contains the information needed to export an ingest pipeline.
-type IngestPipeline struct {
```

Comment:
A lot of this functionality was moved to the `internal/elasticsearch` package since it shares a lot of retrieval needs with the new export functionality